OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -28,6 +28,7 @@ public Hotkey Key
 		}
 
 		public bool DisableKeyRepeat = false;
+		public bool PlaySoundOnKeyPress = true;
```

Comment:
How about `DisableKeySound` to match `DisableKeyRepeat`?  Would that be too unclear?